.analysis_options

```diff
@@ -1,2 +1,4 @@
 analyzer:
-  strong-mode: true
+  strong-mode:
+    implicit-casts: False
+    implicit-dynamic: False
```

Comment:
Does it imply that `dynamic` annotations in `fromUri` and `prettyUri` should be also removed?

When some other package depends on `path` and has `implicit-dynamic: False` analysis option, DDC fails if `dynamic` annotations are omitted:
```
Error compiling dartdevc module:path|lib/lib__path.js

[error] Missing parameter type for 'uri'. (package:path/path.dart, line 387, col 16)
[error] Missing parameter type for 'uri'. (package:path/path.dart, line 433, col 18)
[error] Missing parameter type for 'uri'. (package:path/src/context.dart, line 961, col 18)
[error] Missing parameter type for 'uri'. (package:path/src/context.dart, line 1019, col 20)

Please fix all errors before compiling (warnings are okay).
Build failed.
```

I'm not sure whether it's DDC or style guide issue.